### PR TITLE
fix: Windows interop

### PR DESCRIPTION
**Pull request by richardschneider**
Get running on windows.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,9 +10,9 @@
   "scripts": {
     "lint": "aegir lint",
     "build": "aegir build",
-    "test": "aegir test",
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser",
+    "test": "aegir test --timeout 10000",
+    "test:node": "aegir test --target node --timeout 10000",
+    "test:browser": "aegir test --target browser --timeout 10000",
```
Comment (diasdavid) [inline package.json]: Please avoid using global timeouts. Try first here updating to latest Aegir and then seeing if you still need the timeouts.

**Comment by richardschneider** [comment]
@diasdavid [build](https://travis-ci.org/ipld/js-ipld-resolver/builds/297841521?utm_source=github_status&utm_medium=notification) is failing with new latest aegir and no timeout specified.  Shall I revert, or do u have another idea.

File: package.json [hunk 2]
```diff
@@ -46,10 +46,10 @@
   "dependencies": {
     "async": "^2.5.0",
     "cids": "~0.5.2",
-    "interface-datastore": "^0.3.1",
+    "interface-datastore": "^0.4.1",
     "ipfs-block": "~0.6.0",
     "ipfs-block-service": "~0.12.0",
-    "ipfs-repo": "~0.17.0",
+    "ipfs-repo": "^0.18.0",
```
Comment (diasdavid) [inline package.json]: We avoid using ^ for deps that are < 1.0.0. Keep `~`

File: package.json [hunk 3]
```diff
@@ -46,10 +46,10 @@
   "dependencies": {
     "async": "^2.5.0",
     "cids": "~0.5.2",
-    "interface-datastore": "^0.3.1",
+    "interface-datastore": "^0.4.1",
```
Comment (diasdavid) [inline package.json]: Seems we missed this one

**Comment by diasdavid** [comment]
@richardschneider you can increase the timeout of that specific batch of tests. I just meant to avoid global timeouts, it is important that we know which tests are really slow.

File: package.json (hunk 3 above)
Comment (richardschneider) [reply package.json]: and it should use ~

File: src/index.js [hunk 4]
```diff
@@ -215,9 +220,7 @@ class IPLDResolver {
     if (!r) {
       return callback(new Error('No resolver found for codec "' + options.format + '"'))
     }
-    // TODO add support for different hash funcs in the utils of
-    // each format (just really needed for CBOR for now, really
-    // r.util.cid(node1, hashAlg, (err, cid) => {
```
Comment (diasdavid) [inline src/index.js]: Please bring back this TODO comment

Comment (richardschneider) [reply src/index.js]: done

**Comment by richardschneider** [comment]
@diasdavid ready for review

**Pull request merged**